Frontend/timecamp/src/Components/Fields.jsx

```diff
@@ -1,419 +0,0 @@
-  }
-`;
-
-const CardBody = styled.div`
-  margin: 0 !important;
-  padding: 1rem 0 0;
-  flex: 1 1 auto;
-  background-color: #f3f3f3;
-  & p {
-    text-align: left;
-    margin-top: 0;
-    margin-bottom: 1rem;
-  }
-`;
-const XYZ = styled.div`
-  border-bottom: 0;
-  background-color: #f3f3f3;
-  padding: 0;
-  & h3 {
-    padding: 0;
-    text-align: left;
-    margin-bottom: 0 !important;
-    letter-spacing: 0;
-    cursor: default;
-    font-family: lato, sans-serif;
-    font-size: 1.75rem;
-    font-weight: 500;
-    line-height: 1.1;
-    color: inherit;
-    margin-top: 0;
-  }
-  & button {
-    font-size: 15px;
-    border-bottom: none;
-    text-decoration: none;
-    color: #767676;
-    font-weight: 400;
-    padding-bottom: 10px;
-  }
-`;
-
-const GreenBtn = styled.a`
-  display: flex;
-  margin-top: 2rem;
-  width: fit-content;
-  background-color: #25cf60;
-  border-radius: 26px;
-  padding: 15px 40px;
-  color: #fff;
-  font-weight: 700;
-  text-decoration: none;
-
-  :hover {
-    background-color: #1da44c;
-  }
-`;
-
-export const Fields = () => {
-  return (
-    <div>
-      <FeatureSection>
-        <Container>
-          <Row>
-            <TextDiv>
-              <div>
-                <GP>for employees</GP>
-                <H2>Track your employees working time</H2>
-                <Para>
-                  Log working hours of your employees automatically or manually
-                  and make sure they are billed fairly.
-                </Para>
-                <Content>
-                  <Card>
-                    <div>
-                      <H3>
-                        <button>
-                          Check logged time on the handy Timesheet
-                        </button>
-                      </H3>
-                    </div>
-                    <CardBody>
-                      <p>
-                        Enjoy the transparency - check your employees time
-                        records with more detailed information on the handy
-                        timesheet or a graphical, calendar-like view.
-                      </p>
-                    </CardBody>
-                  </Card>
-
-                  <Card>
-                    <XYZ>
-                      <h3>
-                        <button>Try our easy-to-use desktop app</button>
-                      </h3>
-                    </XYZ>
-                  </Card>
-
-                  <Card>
-                    <XYZ>
-                      <h3>
-                        <button>Track your activities automatically</button>
-                      </h3>
-                    </XYZ>
-                  </Card>
-                </Content>
-                <GreenBtn><Link to="/project/task">Track employees time</Link></GreenBtn>
-              </div>
-            </TextDiv>
-            <ImgDiv>
-              <img
-                src="https://cdn-m.timecamp.com/img/greenbranding/features/img-home-features.png"
-                alt=""
```

Comment:
dont keep alt as empty, but any chance, if image image is not loading, alt text will be visible